lib/panomosity/xlsx_writer.rb

```diff
@@ -0,0 +1,51 @@
+require 'write_xlsx'
+
+module Panomosity
+  module XLSX_Writer
+    def csv_to_xlsx(input_files, output_file)
+      logger.info "Creating #{output_file}.xlsx"
+
+      workbook = WriteXLSX.new("#{output_file}.xlsx")
+
+      if input_files.is_a? Array
```

Comment:
In this repo, we use parentheses here.

https://github.com/elevatesystems/panomosity/pull/4#discussion_r497223367

Reply:
%ThatGuyJosh